fastplotlib/graphics/_features/_positions_graphics.py

```diff
@@ -182,6 +182,24 @@ def set_value(self, graphic, value: float | int):
         self._call_event_handlers(event)
 
 
+# manages the coordinate space for scatter/line
+class SizeSpace(GraphicFeature):
+    def __init__(self, value: str):
+        self._value = str(value)
+        super().__init__()
+
+    @property
+    def value(self) -> str:
+        return self._value
+
+    def set_value(self, graphic, value: str):
+        graphic.world_object.material.size_space = str(value)
```

Comment:
`thickness_space` if the material is for a line, can have a conditional like `if "Line" in material.__class__.__name__`

Reply:
also don't try to coerce `value` to a string, user should give a string